docs/examples/llama_datasets/uploading_llama_dataset.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<p>so if we use github LFS is there any constraints?</p><p>if the dataset is big, can we let them upload to shared s3/google drive/dropbox instead?</p><p>ideal place could be s3 but there might need some configuration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9165/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Github LFS has a file size (i.e per image) restriction of up to 5GB. Would that be enough for most cases? I'd imagine we'd compress these images tho.

The thing with s3 or those other services we need to likely stand up a service, as we probably don't want open write access to our cloud storage.

Reply:
ic thanks! that should be great to start with